### PR TITLE
Undo require form fields

### PR DESCRIPTION
**Pull request by laritakr**
References[ bulkrax 538](https://github.com/samvera-labs/bulkrax/issues/538). Hidden fields cannot be required.

reverse https://github.com/samvera-labs/bulkrax/pull/575 to be handled at a later point.

### CODE REVIEW

**Comment by alishaevn** [comment]
ref https://github.com/samvera-labs/bulkrax/issues/538

**Comment by bkiahstroud** [comment]
Not saying this needs to happen in this PR, but I think a potential solution might be disabling the hidden fields. If the inputs are disabled, no params are sent, and I think the requirement is disabled

**Comment by alishaevn** [comment]
> Not saying this needs to happen in this PR, but I think a potential solution might be disabling the hidden fields. If the inputs are disabled, no params are sent, and I think the requirement is disabled

thanks kiah! larita and selam did have an idea of how to solve this, but this is holding up the qa of 3 other tickets on pals so I advised to unblock our qa first, and then come back to trying to make this work. 🙂

**Review by bkiahstroud — APPROVED** [review]

**Pull request merged**
